docs/easybuild-v5/proposed-changes.md

```diff
@@ -1,9 +0,0 @@
-# Proposed changes for EasyBuild v5.0
```

Comment:
I'm fine with no longer linking to this page, but ideally we don't remove it, and just put pointers to the new location where this info is available (I really dislike hitting broken URLs that were working before)

Reply:
I suggest redirecting it to the EB5 index instead: 5495612